restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -1727,11 +1816,15 @@ private void outboundToPstn(final CreateCall request, final ActorRef sender) thr
 
 
     private void outboundToSip(final CreateCall request, final ActorRef sender) throws ServletParseException {
+        final String uri = (request.getOutboundProxy()!= null && (!request.getOutboundProxy().isEmpty())) ? request.getOutboundProxy() : "";
```

Comment:
@gvagenas @abdulazizali77 is correct. There is 2 different requirements in the doc :

1. Allow to configure the outbound proxy per Account

2. Allow to rewrite the host part of the URL so from a user standpoint they don't have to know the details of through which hops the route will be going through or if we are interconnected directly or not with that route.